Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -844,30 +1090,361 @@ def lift_host_containment_command():
+            'Name': resource.get('name'),
+            'Permission': resource.get('permission_type'),
+            'SHA256': resource.get('sha256'),
+        }
+        file_standard_context = {
+            'Type': resource.get('file_type'),
+            'Name': resource.get('name'),
+            'SHA256': resource.get('sha256'),
+            'Size': resource.get('size'),
+        }
+
+        human_readable = tableToMarkdown(f'CrowdStrike Falcon file {file_id}', file_)
+
+        entry_context = {
+            'CrowdStrike': {
+                'File(val.ID === obj.ID)': file_,
+            },
+            outputPaths['file']: file_standard_context
+        }
+
+        file_content = resource.get('content')
+        if file_content:
+            demisto.results(
+                fileResult(
+                    resource.get('name'),
+                    file_content
+                )
+            )
+
+        return create_entry_object(contents=response, ec=entry_context, hr=human_readable)
+    else:
+        return 'No file found.'
+
+
+def list_files_command():
+    response = list_files()
+
+    resources: list = response.get('resources', [])
+
+    files_output = []
+    file_standard_context = []
+
+    for resource in resources:
+        files_output.append({
+            'ID': resource.get('id'),
+            'CreatedBy': resource.get('created_by'),
+            'CreatedTime': resource.get('created_timestamp'),
+            'Description': resource.get('description'),
+            'Type': resource.get('file_type'),
+            'ModifiedBy': resource.get('modified_by'),
+            'ModifiedTime': resource.get('modified_timestamp'),
+            'Name': resource.get('name'),
+            'Permission': resource.get('permission_type'),
+            'SHA256': resource.get('sha256'),
+        })
+        file_standard_context.append({
+            'Type': resource.get('file_type'),
+            'Name': resource.get('name'),
+            'SHA256': resource.get('sha256'),
+            'Size': resource.get('size'),
+        })
+
+    human_readable = tableToMarkdown('CrowdStrike Falcon files', files_output)
+
+    entry_context = {
+        'CrowdStrike': {
+            'File(val.ID === obj.ID)': files_output
+        },
+        outputPaths['file']: file_standard_context
+    }
+
+    return create_entry_object(contents=response, ec=entry_context, hr=human_readable)
+
+
+def run_script_command():
+    args = demisto.args()
+    script_name = args.get('script_name')
+    raw = args.get('raw')
+    host_ids = argToList(args.get('host_ids'))
+
+    if script_name and raw:
+        raise ValueError('Only one of the arguments script_name or raw should be provided, not both.')
+    elif not script_name and not raw:
+        raise ValueError('One of the arguments script_name or raw must be provided, none given.')
+    elif script_name:
+        full_command = f'runscript -CloudFile={script_name}'
```

Comment:
do we need to support names with spaces?

Reply:
what do you mean?